Wordle.Api/Wordle.Api/Wordle.Api/Data/DailyCompletions.cs

```diff
@@ -4,6 +4,6 @@ namespace Wordle.Api.Data
     public class DailyCompletions
     {
         public int DateWordId { get; set; }
-        public int PlayerId { get; set; }
+        public string PlayerName { get; set; } = "";
```

Comment:
This seems to be from the previous assignment so I'm not gonna comment much on these kinds of things.

Reply:
Fair enough, appreciated!